WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -88,6 +91,9 @@ class ReaderDiscoverViewModel @Inject constructor(
     private val _preloadPostEvents = MediatorLiveData<Event<PreLoadPostContent>>()
     val preloadPostEvents: LiveData<Event<PreLoadPostContent>> = _preloadPostEvents
 
+    private val _sortingType = MutableLiveData<DiscoverSortingType>(POPULARITY)
```

Comment:
Minor 🔍: (IDE warning) Explicit type argument `DiscoverSortingType` can be removed .

```suggestion
    private val _sortingType = MutableLiveData(DiscoverSortingType.POPULARITY)
```

Reply:
Fixed in 860c8f3